Italian/guardare/libri-francesi/libri-francesi.md

```diff
@@ -6,3 +6,7 @@ I volumi sono sistemati con ordine maniacale, in rigoroso ordine alfabetico.
 Muovendo gli occhi lungo i diversi scaffali, incontri un libro stranamente non ricoperto di polvere, come se fosse
 stato letto da poco: "Les histoires horribles d'enfants chauves-souris", che suona qualcosa come "Le terribili storie
 dei bambini pipistrello".
+
+[Lasci perdere i libri e decidi di esaminare le fotografie, sperando di riconoscere qualcuno](../vecchie-fotografie/vecchie-fotografie.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.